nebula_decoders/include/nebula_decoders/nebula_decoders_continental/decoders/continental_ars548_decoder.hpp

```diff
@@ -35,36 +34,22 @@ namespace drivers
 namespace continental_ars548
 {
 /// @brief Continental Radar decoder (ARS548)
-class ContinentalARS548Decoder : public ContinentalPacketsDecoder
+class ContinentalArs548Decoder : public ContinentalPacketsDecoder
```

Comment:
I think typically we have either alllower case (for snake) or upper case for model letters, as in this case. Did you find a precedent that led to this change?

Reply:
The precedent is that ros makes the following conversion at times `ContinentalArs548` <-> `continental_ars_548`, so I though to change it to Ars548 to avoid  ARS548 converted to `a_r_s_548`.
In the past LiDARTag was transformed to `li_d_a_r_tag` which is now everything there is called `lidartag`

Reply:
OK, makes sense - is this also going to affect, for example, AT128?

Reply:
Since only the continental drivers define new messages (the part in which ros does the previously mentioned processing) it should not affect Hesai as of now. If you prefer I can change only the class names back to ARS548/SRR520. It takes about 5m

Reply:
Hmm, yes I think the class names would be good to be consistent with other decoders.
Sorry!

Reply:
Addressed in [1dea5f5](https://github.com/tier4/nebula/pull/151/commits/1dea5f5ce3fab83249e3adb600bae540da11ceaa)
Note that message classes are still with the Ars548 naming

Reply:
Thank you!